### ISSUE TITLE
Announce zoom

### ISSUE THREAD

**Issue by prushforth** [Maps4HTML/Web-Map-Custom-Element#664]
Per "2 [The zoom level should be announced](https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/396#issue-858020120)", we should make it so that the zoom level is always announced when it changes.

**Comment by prushforth** [comment]
- [ ] Update Web-Map-Custom-Element to audibly render zoom in output by default i.e. on by default
- [ ] Update MapML-Extension to reflect the fact that the option to audibly render zoom is on by default, provide option to turn it off.  "Announce movement for screen readers" -> "Announce Zoom"
- [ ] Update Web-Map-Doc.
- [ ] Change the name of [this experiment](https://maps4html.org/experiments/screenreader/announce-pan-zoom/).  We should add the pan/zoom control to that map, as well as a \<map-caption>Paris, the City of Light\</map-caption>
